README.md

```diff
@@ -558,6 +559,23 @@ findCapitalByProvince("آذربایجان شرقی"); // تبریز
 findCapitalByProvince("دبی");
 ```
 
+### Validate Iranian Postal code
+
+**Usage**
+
+> this function validate entry postal code and return boolean for validation confirm
```

Comment:
Replace with `This function validates the entry postal code and returns a boolean for validation confirmation.`